unreleased/changelog.yaml

```diff
@@ -0,0 +1,15 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```

Comment:
Please name this file something more unique, such as the name of your branch.

Reply:
thx, very useful